packages/opentelemetry-metrics/src/Handle.ts

```diff
@@ -130,12 +130,44 @@ export class GaugeHandle extends BaseHandle implements types.GaugeHandle {
  * MeasureHandle is an implementation of the {@link MeasureHandle} interface.
  */
 export class MeasureHandle extends BaseHandle implements types.MeasureHandle {
+  constructor(
+    labelSet: types.LabelSet,
+    private readonly _disabled: boolean,
```

Comment:
if something is readonly and disabled, is there no way to enable it? How is this different than noop

Reply:
It seems like it is just a way for sdk to create noop handles via ctor, but I'm not sure if there are any benefits to it over just having `NoopHandle` or something else. It is a lot less code with just `_disabled`! Maybe @mayurkale22 would know?